### ISSUE TITLE
Sticky drawer items disappear on profile item selected

### ISSUE THREAD

**Issue by iraupph** [mikepenz/MaterialDrawer#348]
Hi, I'm having a problem that whenever I select any item from the profile list (profile or setting) all of my sticky drawer items (one or more) disappear. My setup is the following:
```java
...
private AccountHeader buildDrawerAccountHeader(Bundle savedInstanceState) {
        return new AccountHeaderBuilder()
                .withActivity(this)
                .withCompactStyle(true)
                .withProfileImagesVisible(false)
                .withHeaderBackground(new ColorDrawable(getResources().getColor(R.color.accent_material_purple_A700)))
                .addProfiles(
                        new ProfileDrawerItem().withName("Ícaro Raupp").withIcon(new IconicsDrawable(this, GoogleMaterial.Icon.gmd_person)),
                        new ProfileSettingDrawerItem().withName("Logout").withIcon(new IconicsDrawable(this, GoogleMaterial.Icon.gmd_exit_to_app)).withIdentifier(C.ACCOUNTS_LOG_OUT)
                )
                .withSavedInstance(savedInstanceState)
                .build();
    }
...
private Drawer buildDrawer(Bundle savedInstanceState, Toolbar toolbar) {
        return new DrawerBuilder(this)
                .withActivity(this)
                .withToolbar(toolbar)
                .withRootView(R.id.frame_container) // Drawer will be inside this view
                .withActionBarDrawerToggleAnimated(true)
                .withAccountHeader(mDrawerAccountHeader)
                .addDrawerItems(
                        new PrimaryDrawerItem().withName("Home").withIcon(GoogleMaterial.Icon.gmd_home),
                        new PrimaryDrawerItem().withName("List").withIcon(GoogleMaterial.Icon.gmd_list),
                        new SectionDrawerItem().withName("Categories"),
                        new SecondaryDrawerItem().withName("Education").withIcon(GoogleMaterial.Icon.gmd_book)
                )
                .addStickyDrawerItems(
                        new SecondaryDrawerItem().withName("Settings").withIcon(GoogleMaterial.Icon.gmd_settings).withIdentifier(C.DRAWER_SETTINGS)
                )
                .withSavedInstance(savedInstanceState)
                .build();
    }
...
```
Here's are some screen shots of the behaviour:
![Steps](https://cloud.githubusercontent.com/assets/1833584/7790074/e4fef094-024e-11e5-910a-be64e3d55305.png)

I was mostly referencing the [Complex Header Drawer](https://github.com/mikepenz/MaterialDrawer/blob/master/app/src/main/java/com/mikepenz/materialdrawer/app/ComplexHeaderDrawerActivity.java) sample for this.
Any ideas what could be the problem?

Thanks

**Comment by mikepenz** [comment]
@iraupph i am not sure when you call the methods from above. but you should only call them in the onCreate method. As these are only required when creating the view.

**Comment by iraupph** [comment]
Hi, I am calling them like this:
```java
...
    // Class fields
    private AccountHeader mDrawerAccountHeader;
    private Drawer mDrawer;
...
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        // Define this Toolbar as the ActionBar for backwards compatibility
        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);

        mDrawerAccountHeader = buildDrawerAccountHeader(savedInstanceState);
        mDrawer = buildDrawer(savedInstanceState, toolbar);
    }
...
   // I also have this
    @Override
    protected void onSaveInstanceState(Bundle outState) {
        outState = mDrawer.saveInstanceState(outState);
        outState = mDrawerAccountHeader.saveInstanceState(outState);
        super.onSaveInstanceState(outState);
    }
```

Yes, the sample works without this happening which is even stranger D:

**Comment by mikepenz** [comment]
@iraupph i just tried it with the customContainer sample added a header. and it was working too. 

I have really no idea why it is not working for you. do you modify the drawerItems somewhere else? any modifications?

**Comment by iraupph** [comment]
Quick update, I just figured something out, after it disappearing, if I open the drawer again and just show and hide the account list, the sticky header shows up again like this:
![problem1](https://cloud.githubusercontent.com/assets/1833584/7797622/1cf18bc6-02c5-11e5-8c0f-b898947590b8.gif)

Maybe the sticky items are being pushed down outside of the visible drawer or something?

**Comment by mikepenz** [comment]
@iraupph haha good one :) i think i'm now able to hunt down the issue. :P thank you

**Comment by mikepenz** [comment]
@iraupph wait 30 minutes. this is fixed in v3.0.2

**Comment by iraupph** [comment]
Nice, thanks!
